### PR TITLE
Implement Threads overview container [full flow]

### PR DESCRIPTION
**Pull request by montakarlo**

### CODE REVIEW

File: frontend/src/services/threadsService.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import api from '../common/helpers/apiHelper';
 import { ICreateComment } from 'common/models/post/ICreateComment';
+import { IPostsForThreads } from 'common/models/post/IPostsForThreads';
 
-export const getThreads = async () => {
-  const response = await api.get('/threads');
+export const getThreads = async ({ userId, activeWorkspaceId }: IPostsForThreads) => {
+  const response = await api.get(`/api/posts/${userId}/${activeWorkspaceId}/threads`);
```
Comment (AlikRakhmonov) [inline frontend/src/services/threadsService.ts:6]: api is incorrect

File: backend/src/api/routes/postRoutes.ts [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 import { run } from '../../common/utils/routeHelper';
-import { addPost, editPost, addComment, getPostComments } from '../../services/postService';
+import { addPost, editPost, addComment, getPostComments, getPostsByUserId } from '../../services/postService';
 
 const router = Router();
 
 router
   .get('/:id/comments', run((req: Request) => getPostComments(req.params.id)))
   .post('/:id/comments', run((req: Request) => addComment(req.user.id, req.params.id, req.body)))
+  .get('/:id/:activeworkspaceid/threads', run((req: Request) => getPostsByUserId(req.params.id, req.params.activeworkspaceid)))
```
Comment (AlikRakhmonov) [inline backend/src/api/routes/postRoutes.ts:10]: you don't follow the rules! It should be REST. We do not have threads in business logic

File: frontend/src/services/threadsService.ts (hunk 1 above)
Comment (montakarlo) [reply frontend/src/services/threadsService.ts:6]: which api I have to create?

File: frontend/src/containers/Thread/index.tsx [hunk 3]
```diff
@@ -24,27 +26,28 @@ interface IProps {
 }
 
 const Thread: FunctionComponent<IProps> = ({
+  showOnlyTwoComments = false,
+  chatName = '',
   width = 'auto',
+  maxThreadHeight = '600px',
```
Comment (AlikRakhmonov) [inline frontend/src/containers/Thread/index.tsx:32]: 600px is incorrect for the default value

File: backend/src/api/routes/postRoutes.ts (hunk 2 above)
Comment (AlikRakhmonov) [reply backend/src/api/routes/postRoutes.ts:10]: activeworkspaceid why is it lowercase?

File: backend/src/data/repositories/userRepository.ts [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import { EntityRepository, Repository } from 'typeorm';
 import { User } from '../entities/User';
+// import { Chat } from '../entities/Chat';
```
Comment (yklym) [inline backend/src/data/repositories/userRepository.ts:3]: comment

File: frontend/src/components/ProfilePreview/index.tsx [hunk 5]
```diff
@@ -84,6 +84,7 @@ const ProfilePreview: FunctionComponent<IProps> = ({ user, onSend, openProfile }
         className={styles.link}
       >
         <img
+          style={{ objectFit: 'cover' }}
```
Comment (yklym) [inline frontend/src/components/ProfilePreview/index.tsx:87]: Not okay to use styles like this

File: frontend/src/sagas/index.ts [hunk 6]
```diff
@@ -15,6 +16,7 @@ export default function* rootSaga() {
     workspaceSaga(),
     invitePopupSaga(),
     joinInvitedWorkspaceSaga(),
+    threadsSaga(),
     threadSaga()
   ]);
```
Comment (yklym) [inline frontend/src/sagas/index.ts:21]: Critical bug, nice

**Pull request merged**